docs/source/actions.rst

```diff
@@ -808,6 +808,38 @@ pack:
 
     st2 run core.http url="http://httpbin.org/get" method="GET" username=user1 password=pass1
 
+* ``core.announcement`` : Action that broadcasts the announcement to all stream consumers.
```

Comment:
General, as the existing actions started the sentences with "This action", I suggest the extra ones keep the same wording, e.g. "This action allows broadcast of accouncement". Or update the two previous action so that they read the same way.